lib/app-capitano.coffee

```diff
@@ -33,8 +33,10 @@ capitano.globalOption
 	boolean: true
 	alias: 'h'
 
-# ---------- Info Module ----------
-capitano.command(actions.info.version)
+capitano.globalOption
+	signature: 'version'
+	boolean: true
+	alias: 'v'
```

Comment:
It's the pre-parser in `app.ts` and then oclif (rather than Capitano) that handles `balena --version`, but this changes the output of `balena --help` which is handled by Capitano:
```
$ balena --help
...
Global Options:
    --help, -h
    --version, -v
```